src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1697,8 +1697,9 @@ void ApplicationManagerImpl::RemoveHMIFakeParameters(
   factory.attachSchema(*message, true);
 }
 
-bool ApplicationManagerImpl::Init(resumption::LastState& last_state,
-                                  media_manager::MediaManager* media_manager) {
+bool ApplicationManagerImpl::Init(
```

Comment:
@LitvinenkoIra Is a part of public interface?

Reply:
@AByzhynar Yes. And method marked as deprecated https://github.com/smartdevicelink/sdl_core/pull/2626/files#diff-21171d7cbfd36d53d3224063bd66b757R124 
(take a look at application_manager.h)

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508